dip-0006/bls_signature_scheme.md

```diff
@@ -110,23 +110,26 @@ We chose to enforce distinct messages when signing with a secret key. Dash’s i
 
 ## BLS Curves used
 
-The BLS signature scheme requires pairing-friendly curves to be used. Multiple curves with different security, performance and space characteristics have been proposed by different parties. At the time this document was written, no decision had been made as to which curve would be used.
-
-However, the choice of the specific curve should have no influence on the scheme itself, how BLS is used or what is possible.
-
-This document will be updated when a decision has been made on which curve to use.
+The BLS signature scheme requires pairing-friendly curves to be used, of which only a few suitable exist. Dash is
+using the BLS12-381 with reversed G1 and G2 groups. A specification can be found in the Chia [bls-signatures](https://github.com/Chia-Network/bls-signatures/blob/master/SPEC.md)
+repository.
 
 ## Serialization format of BLS primitives
 
-It is necessary to serialize BLS primitives into a byte representation so that they can be used inside other serialized data structures. How this is performed depends on the curves and libraries used. This has not been defined yet and this document will be updated later.
+It is necessary to serialize BLS primitives into a byte representation so that they can be used inside other serialized data structures.
+The serialization format is described in the Chia [bls-signatures](https://github.com/Chia-Network/bls-signatures/blob/master/SPEC.md)
+repository.
```

Comment:
Is there a reason for the line breaks here?

Reply:
It's very hard to read/edit otherwise in my editor. In Markdown, single-newlines are ignored when rendered, so it's the same as if there was no newline.

I'd suggest to limit line width in the future when we create DIPs to make editing/reading the plain Markdown easier.

Reply:
We have this problem in all DIPs and nearly all paragraphs are in a single line...

Reply:
I don't see it as a "problem" since editors can be made to soft fold, _but_, I also understand and think if we do this then let's set at something standard like 72 or 80 and hard-wrap at that then.